src/main/java/org/folio/list/services/export/CsvCreator.java

```diff
@@ -80,7 +82,15 @@ public ExportLocalStorage createAndUploadCSV(ExportDetails exportDetails, String
       ContentsRequest contentsRequest = new ContentsRequest().entityTypeId(list.getEntityTypeId())
         .fields(exportDetails.getFields())
         .ids(ids);
-      var sortedContents = queryClient.getContents(contentsRequest);
+      var sortedContents = queryClient.getContents(contentsRequest)
+        .stream()
+        .filter(map -> !(map.containsKey(IS_DELETED) && (boolean) map.get(IS_DELETED)))
```

Comment:
!contains || is false
```suggestion
        .filter(map -> !Boolean.TRUE.equals(map.get(IS_DELETED)))
```
This will ensure it is not true (false or `null`, in the case that it doesn't exist)